test/fixtures/cmd/http-windows-remote-no-options

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
Sadly, this file appears to be invalid JSON. It looks like there is a mismatched curly bracket in the CSS.